roles/jenkins_builder/tasks/jenkins_credential.yml

```diff
@@ -0,0 +1,6 @@
+---
+- name: Placing ssh public key for Jenkins_int_rht
```

Comment:
This needs to be included into main.yml. Ansible will only run main.yml by itself.